File: src/main/java/org/redrune/net/packet/context/impl/PingPacketContext.java
package org.redrune.net.packet.context.impl;

import org.redrune.game.entity.actor.player.Player;
import org.redrune.net.packet.context.PacketContext;

/**
 * @author Tyluur <itstyluur@icloud.com>
 * @since 2019-02-04
 */
public class PingPacketContext extends PacketContext {
	
	/**
	 * The latency between client and server
	 */
	private final int ping;
	
	public PingPacketContext(int ping) {
		this.ping = ping;
	}
	
	@Override
	public void handle(Player player) {
	
	}
}

File: src/main/kotlin/org/redrune/networking/packet/context/impl/PingPacketContext.kt
package org.redrune.networking.packet.context.impl

import org.redrune.game.entity.actor.player.Player
import org.redrune.networking.packet.context.PacketContext

/**
 * @author Tyluur <itstyluur@icloud.com>
 * @since 2019-02-04
 */
class PingPacketContext(
    /**
     * The latency between client and server
     */
    private val ping: Int,
) : PacketContext() {
    override fun handle(player: Player) {}
}